integration/query_evalution_test.go

```diff
@@ -0,0 +1,164 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func TestEvalMod(t *testing.T) {
+	//	t.Parallel()
```

Comment:
It might be easier to run tests with `go test -parallel=1 run=TestEvalMod` than do temporary modifications in code